.github/workflows/build.yml

```diff
@@ -481,11 +481,15 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
```

Comment:
Is this just for obtaining the version of TFIO? Perhaps a simple bash would be more reasonable than requiring a specific version of Python?

Reply:
@terrytangyuan any python3 version should address the issue I suppose (need to check after merging). I chose 3.8 as it was the latest. 

Also, if we are using bash to get the version, we would have to grep for version in the `tensorflow_io/core/python/ops/version_ops.py` file and perform some split operations. I felt just fetching the version from setup.py would be simpler. Let me know if I am missing anything. Thanks.